cmd/copy.go

```diff
@@ -1652,6 +1652,9 @@ func (cca *CookedCopyCmdArgs) processCopyJobPartOrders() (err error) {
 	if err != nil {
 		if err == NothingToRemoveError || err == NothingScheduledError {
 			return err // don't wrap it with anything that uses the word "error"
+		} else if cca.S2sPreserveBlobTags {
```

Comment:
Shouldn't we check if the preserve blob tags flag is enabled **and** the error message is the AuthorizationPermissionMismatch?

Reply:
We are doing exactly the same thing here in line no 1655